### PR TITLE
weather app

### PR DESCRIPTION
**Pull request by NabilFarhat**

### CODE REVIEW

File: weatherApp/ViewController.swift [hunk 1]
```diff
@@ -10,11 +10,64 @@ import UIKit
...
+        
+        let  task = URLSession.shared.dataTask(with: url!) { (data, respose, error) in
+            if error == nil {
+                
+                if let unWrappedData = data {
+                    do
+                    {   // parsing the response into json object
+                        let jsonResult = try JSONSerialization.jsonObject(with: unWrappedData, options:  JSONSerialization.ReadingOptions.mutableContainers) as? NSDictionary
+                        
+                        // print(jsonResult)
```
Comment (pandurangsyt) [inline weatherApp/ViewController.swift]: cleanup the dead code

File: weatherApp/ViewController.swift [hunk 2]
```diff
@@ -10,11 +10,64 @@ import UIKit
...
+                        let weather = jsonResult?["weather"] as? NSArray
+                        let weatherItem = weather?[0] as! NSDictionary
+                        let description = weatherItem["description"] as! String
+                        
+                   
+                        
+                        DispatchQueue.main.async {
+                             self.resultLabel.text = description
+                        }
+
```
Comment (pandurangsyt) [inline weatherApp/ViewController.swift]: remove extra spaces, empty line

File: weatherApp/ViewController.swift (hunk 1 above)
Comment (NabilFarhat) [reply weatherApp/ViewController.swift]: fixed it

File: weatherApp/ViewController.swift (hunk 2 above)
Comment (NabilFarhat) [reply weatherApp/ViewController.swift]: fixed it

**Pull request merged**